sql/moz-fx-data-shared-prod/firefox_desktop/baseline_active_users/view.sql

```diff
@@ -0,0 +1,63 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_desktop.baseline_active_users`
+AS
+SELECT
+  *,
+  CASE
+    WHEN isp = 'BrowserStack'
+      THEN CONCAT('Firefox Desktop', ' ', isp)
+    WHEN distribution_id = 'MozillaOnline'
+      THEN CONCAT('Firefox Desktop', ' ', distribution_id)
+    ELSE 'Firefox Desktop'
+  END AS app_name,
+  CASE
+    WHEN BIT_COUNT(days_active_bits)
+      BETWEEN 1
+      AND 6
+      THEN 'infrequent_user'
+    WHEN BIT_COUNT(days_active_bits)
+      BETWEEN 7
+      AND 13
+      THEN 'casual_user'
+    WHEN BIT_COUNT(days_active_bits)
+      BETWEEN 14
+      AND 20
+      THEN 'regular_user'
+    WHEN BIT_COUNT(days_active_bits) >= 21
+      THEN 'core_user'
+    ELSE 'other'
+  END AS activity_segment,
+  CASE
+    WHEN BIT_COUNT(days_desktop_active_bits)
+      BETWEEN 1
+      AND 6
+      THEN 'infrequent_user'
+    WHEN BIT_COUNT(days_desktop_active_bits)
+      BETWEEN 7
+      AND 13
+      THEN 'casual_user'
+    WHEN BIT_COUNT(days_desktop_active_bits)
+      BETWEEN 14
+      AND 20
+      THEN 'regular_user'
+    WHEN BIT_COUNT(days_desktop_active_bits) >= 21
+      THEN 'core_user'
+    ELSE 'other'
+  END AS segment_dau,
+  IFNULL(mozfun.bits28.days_since_seen(days_active_bits) = 0, FALSE) AS is_dau,
+  IFNULL(mozfun.bits28.days_since_seen(days_active_bits) < 7, FALSE) AS is_wau,
+  IFNULL(mozfun.bits28.days_since_seen(days_active_bits) < 28, FALSE) AS is_mau,
+  IFNULL(mozfun.bits28.days_since_seen(days_desktop_active_bits) = 0, FALSE) AS is_desktop_dau,
+  IFNULL(mozfun.bits28.days_since_seen(days_desktop_active_bits) < 7, FALSE) AS is_desktop_wau,
+  IFNULL(mozfun.bits28.days_since_seen(days_desktop_active_bits) < 28, FALSE) AS is_desktop_mau,
```

Comment:
@irrationalagent / @kik-kik 
I have added the new fields (kept the existing ones). Does the field names `is_desktop_dau` seem ok?

Reply:
I think we want the columns you have as `is_desktop_dau` to be just `is_dau`, reason being that this is the official definition of "dau" and mirrors what we do currently for legacy telemetry on desktop.

I think we should rename the columns based on `days_active_bits` to be something else. these are based on the `duration` column so maybe `is_dau_duration` could be an option.